services/web-server/src/login/strategies/github.js

```diff
@@ -37,6 +37,12 @@ module.exports = class Github {
     }
 
     const githubClient = this.clients.get(userId);
+
+    if (!githubClient) {
+      debug(`Github client for user id ${userId} could not be found.`);
```

Comment:
I suspect this will be common, since the entry is only added to the map when the user logs in, and it's an in-memory map.  So once web-server restarts, or if the user hits a different process, it won't exist.